src/commands/set.js

```diff
@@ -104,8 +107,15 @@ const setBoolean = (dotNotation, value) => {
 	return setValue(dotNotation, newValue);
 };
 
-const setArrayString = (dotNotation, value, inputs) =>
-	setValue(dotNotation, inputs);
+const setArrayURL = (dotNotation, value, inputs) => {
+	inputs.forEach(input => {
+		const parsedURL = url.parse(input);
+		if (!API_PROTOCOLS.includes(parsedURL.protocol) || !parsedURL.hostname) {
+			throw new ValidationError(URL_ERROR_MESSAGE);
```

Comment:
If someone provides a valid but unsupported protocol (e.g. `ws:`) this message will be misleading. Better would be a message with a list of supported protocols.

Reply:
something like this?
```
Node URLs must include a supported protocol (http,https) and a hostname. E.g. https://127.0.0.1:4000 or http://localhost.
```